src/graph/backend/dnnl/passes/insert_ops.cpp

```diff
@@ -833,10 +844,11 @@ status_t insert_unsqueeze_for_prelu(std::shared_ptr<subgraph_t> &sg) {
         const bool per_channel_broadcast
                 = cur_op->get_attr<bool>(op_attr::per_channel_broadcast);
 
-        if (!prelu_doable(ltw(src_lt).vdims(), ltw(wei_lt).vdims(), data_format,
-                    per_channel_broadcast)) {
-            return status::invalid_shape;
-        }
+        VCHECK_INSERT_OPS(prelu_doable(ltw(src_lt).vdims(), ltw(wei_lt).vdims(),
```

Comment:
I suggest to put `prelu_doable()` into a separate line and pass its return value to `VCHECK_INSERT_OPS`. It will make the code easier to read and debug.

Reply:
changed as suggested. please take a look again. thanks